src/ModelContextProtocol.AspNetCore/McpEndpointRouteBuilderExtensions.cs

```diff
@@ -3,10 +3,11 @@
 using Microsoft.AspNetCore.Routing;
 using Microsoft.Extensions.DependencyInjection;
 using ModelContextProtocol.AspNetCore;
-using ModelContextProtocol.Protocol.Messages;
 using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Builder;
+using ModelContextProtocol.Protocol;
 
-namespace Microsoft.AspNetCore.Builder;
+namespace Microsoft.AspNetCore.Routing;
```

Comment:
@halter73, this one I'm less sure about. Generally we put extensions in the same namespace as the thing they're extending, so as this extends IEndpointRouteBuilder, which is in Routing, I put this in Routing to. The method returns a type in the Microsoft.AspNetCore.Builder namespace, though.

Reply:
```suggestion
namespace Microsoft.AspNetCore.Builder;
```

Let's keep this in Microsoft.AspNetCore.Builder. I know it's not consistent with how we choose namespaces for extension methods in general, but it aligns with MapPost/MapGet, MapControllers, MapRazorPages, MapHub, etc. which all target IEndpointRouteBuilder like MapMcp does.